include/attribute.h

```diff
@@ -0,0 +1,282 @@
+    u8 numElectricFan;
+    u8 numWindScroll;
+    u8 numWindCrystalSegment;
+    u8 numWindCrystal;
+    u8 numBackLight;
+    u8 numShinnyScroll;
+    u8 numBloodSucker;
+    u8 numDarkRustCup;
+    u8 numAccelerateWound;
+    u8 numSuperAccelerator;
+    u8 numLuckScroll;
+    u8 numLuckyLeaf;
+    u8 numToggleSwitch;
+    u8 numEssenceScroll;
+    u8 numBodyArtScroll;
+    u8 numPhysicalModifyBook;
+    u8 numDistributionDevice;
+    u8 numDestroySwitch;
+    u8 numPayaPowder;
+    u8 numPoisonPowder;
+    u8 numSleepPowder;
+    u8 numRustPowder;
+    u8 numForceLicense;
+    u8 numDefendLicense;
+    u8 numSuperManLicense;
+    u8 numLegendWound;
+    u8 numMindEyeGlasses;
+    u8 numMartialTutorBook;
+    u8 numStrongLuckAmulet;
+    u8 numHealthBucket;
+    u8 numLegendBamboo;
+    u8 numOldGrindStone;
+    u8 numRepareParts;
+    u8 numLegendIronString;
+    u8 numDareDevelLoveLetter;
+    u8 numSuperMotor;
+    u16 filler4DE;
+    u8 suitRings[0xB]; // TODO:list name.
+    u8 suitBracelets[0xB];
+    u8 suitShoes[0xB];
+    u8 numMilitantShoe;
+    u8 numEscapeShoe;
+    u8 numNormalExtendRing;
+    u8 numSuperExtendRing;
+    u8 numUltraExtendRing;
+    u8 numSpeedBracelet;
+    u8 numGuardBracelet;
+    u8 numAttackBracelet;
+    u8 numCritEnhanceRing;
+    u8 numMagBracelet;
+    u8 numActiveRing;
+    u8 numAwakeRing;
+    u8 numDetoxifyRing;
+    u8 numGoldenRing;
+    u8 numBossRing1;
+    u8 numBossRing2;
+    u8 numCrafterRing;
+    u8 normalFishPole; //00 = don't have
+    u8 superFishPole;
+    u8 ultraFishPole;
+    u8 chopTutorial;
+    u8 mysteriouStone;
+    u8 summonStone;
+    u8 lotteryHouseBill;
+    u8 lostCat;
+    u8 craftEntrust1;
+    u8 craftEntrust2;
+    u8 craftEntrust3;
+    u8 craftEntrust4;
+    u8 craftEntrust5;
+    u8 craftEntrust6;
+    u8 craftEntrust7;
+    u8 craftEntrust8;
+    u8 craftEntrust9;
+    u8 jetsEntrust;
+    u8 zaksEntrst;
+    u8 edisEntrust;
+    u8 entrustFail1;
+    u8 entrustFail2;
+    u8 entrustFail3;
+    u8 shifusWeapon;
+    u8 mightBook1;
+    u8 mightBook2;
+    u8 mightBook3;
+    u8 mightBook4;
+    u8 mightBook5;
+    u8 craftMightBook;
+    u8 mightBook6; // looks wierd.maybe need a fix
+    u8 auditionMachine;
+    u8 waterDragonScale;
+    u8 inviteToTheGreatWar;
+    u8 watchingWarInvite;
+    u8 ex1Invite;
+    u8 ex2Invite;
+    u8 jvhuaChaInStory;
+    u8 luggageToSend;
+    u8 goldFaction99Entrust;
+    u8 zaksHammer;
+    u8 jetEntrustFail;
+    u8 zaksEntrustFail; // where's edi?
+    u8 filler53D;
+    u8 filler53E;
+    u8 filler53F;
+    u8 filler540[0x24];
+    u8 weaponDex[0x1E]; // This is a bit field. 1 = dex lighted, 0 = not lighted.
+    u8 filler582; // wierd
+    u8 weaponSkillDex1[6];
+    u8 weaponSkillDex2[6]; //maybe a useless checksum, need more evidence
+    u8 weaponSpecialEffectDex1[5];
+    u8 filler594;
+    u8 weaponSpecialEffectDex2[5];
+    u8 filler59A;
+    u8 summonGuardianDex[17];
+    u32 money; // max = 0x0098967F
```

Comment:
not readable. Why not just write `9999999`?

Reply:
I think there must be a sort of function like a CheckMoney(), so listing the hex value here may be convenient ^_^